L1Trigger/TrackFindingTracklet/interface/TrackletLUT.h

```diff
@@ -5,6 +5,8 @@
 #include <vector>
 #include <string>
 
+#include "L1Trigger/TrackTrigger/interface/Setup.h"
```

Comment:
I can not see that you are actually using Setup in the code. You do you tt:SensorModule*. To reduce dependencies you can forward declare this pointer.